lib/type/unit/physicalConstants.js

```diff
@@ -4,69 +4,69 @@ function factory (type, config, load, typed, math) {
   // Source: http://www.wikiwand.com/en/Physical_constant
 
   // Universal constants
-  lazy(math, 'speedOfLight',         function () {return type.Unit.parse('299792458 m s^-1')});
-  lazy(math, 'gravitationConstant',  function () {return type.Unit.parse('6.6738480e-11 m^3 kg^-1 s^-2')});
-  lazy(math, 'planckConstant',       function () {return type.Unit.parse('6.626069311e-34 J s')});
-  lazy(math, 'reducedPlanckConstant',function () {return type.Unit.parse('1.05457172647e-34 J s')});
+  lazy(math, 'speedOfLight',         function () {return type.Unit.parse('299792458 m s^-1')._fix()});
```

Comment:
I had something in mind along these lines:

```js
lazy(math, 'speedOfLight',         function () {return fixedUnit('299792458 m s^-1')});
lazy(math, 'gravitationConstant',  function () {return fixedUnit('6.6738480e-11 m^3 kg^-1 s^-2')});
lazy(math, 'planckConstant',       function () {return fixedUnit('6.626069311e-34 J s')});
lazy(math, 'reducedPlanckConstant',function () {return fixedUnit('1.05457172647e-34 J s')});

// ...

// helper function to create a unit with a fixed prefix
function fixedUnit(str) {
  var unit = type.Unit.parse(str);
  unit.fixPrefix = true;
  return unit;
}
```

but this `_fix()` method works ok too. What do you think?